bazel_tools/lsp-types-normalisation.patch

```diff
@@ -35,7 +35,7 @@ index 8970d37..e1a7233 100644
 +        | otherwise = drv
 +
  toNormalizedFilePath :: FilePath -> NormalizedFilePath
- toNormalizedFilePath fp = NormalizedFilePath nuri nfp
+ toNormalizedFilePath fp = normalizedFilePath nuri nfp
```

Comment:
what's the difference between NormalizedFilePath and normalizedFilePath ?

Reply:
None, they just made the newtype opaque and this is the smart constructor but at this point they are equivalent.